enterprise/frontend/src/embedding-sdk/lib/theme/embedding-color-palette.ts

```diff
@@ -35,6 +37,8 @@ export const SDK_TO_MAIN_APP_COLORS_MAPPING: Record<
   (ColorName | NEW_SEMANTIC_COLOR)[]
 > = {
   brand: ["brand"],
+  "brand-hover": ["brand-light"],
+  "brand-hover-light": ["brand-lighter"],
```

Comment:
Follows the convention of `background-hover` mapping to `bg-light`. Because we have `brand-lighter` which is even lighter, we add `brand-hover-light` to compensate for that.